requirements/requirements_build.txt

```diff
@@ -1,4 +1,3 @@
-black==25.1.0
```

Comment:
Hi @jorgepiloto, again, this is used in the `ci/code_generation.py` script. We cannot remove it here or we need to also modify the file mentioned.

Reply:
Hi @PProfizi, I do not see the tool being used in the script you refer to https://github.com/ansys/pydpf-core/blob/master/.ci/code_generation.py

Reply:
@jorgepiloto sorry I meant [here](https://github.com/ansys/pydpf-core/blob/a5384553348b86393a6c169a7872ceb8a407e8e3/src/ansys/dpf/core/operators/build.py#L168), which is what runs when we call `code_generation.py`. @moe-ad FYI

Reply:
We basically use it to reformat some code we autogenerate based on a template.

Reply:
Thanks for explanation, @PProfizi. @moe-ad, what do you think it is possible to explore Ruff's API for this? I am not sure if this tool has an API for Python. If it had, we could completely drop black.